meta/AggregationTable.php

```diff
@@ -176,7 +176,14 @@ protected function renderActiveFilters() {
         $fltrs = array();
         foreach($filters as $column => $filter) {
             list($comp, $value) = $filter;
-            $fltrs[] = $column . ' ' . $comp . ' ' . $value;
+
+            // display the filters in a human readable format
+            foreach ($this->columns as $col) {
+                if ($column === $col->getFullQualifiedLabel()) {
+                    $column = $col->getTranslatedLabel();
+                }
+            }
+            $fltrs[] = "$column '$value'";
```

Comment:
Shouldn't the comparator `$comp` still be used here?

Reply:
I didn't think that `*~` was very user friendly. Can we get rid of it in the filter status message?